### ISSUE TITLE
Please delete an entry in my homebrew

### ISSUE THREAD

**Issue by Wenneker** [stolksdorf/homebrewery#518]
I made a mistake in copy & pasting content - included explicit HTML code into the editor. The result is an entry in the Homebrew listing for my users which includes about 4 pages of content for the name, which also includes an image.

The homebrewry semantec versioning shows v0.0.0 and the delete feature for such isn't working. The edit link likewise won't work to modify the content.

### Additional Details

**Share Link** : http://homebrewery.naturalcrit.com/share/SyKN61q2W
a link - http://homebrewery.naturalcrit.com/edit/ByeYVTJ9h-

**Comment by Wenneker** [comment]
Its showing up as 

> The Homebrewery\r\nWelcome traveler from an antique land. Please sit and tell us of what you have seen. The unheard of monsters, who slither and bite. Tell us of the wondrous items and and artifacts you have found, their mysteries yet to be unlocked. Of the vexing vocations and surprising skills you have seen.\r\n\r\n### Homebrew D&D made easy\r\nThe Homebrewery makes the creation and sharing of authentic looking Fifth-Edition homebrews easy. It uses [Markdown](https://help.github.com/articles/markdown-basics/) with a little CSS magic to make your brews come to life.\r\n\r\n**Try it! **Simply edit the text on the left and watch it *update live* on the right.\r\n\r\n\r\n\r\n### Editing and Sharing\r\nWhen you create your own homebrew you will be given a *edit url* and a *share url*. Any changes you make will be automatically saved to the database within a few seconds. Anyone with the edit url will be able to make edits to your homebrew. So be careful about who you share it with.\r\n\r\nAnyone with the *share url* will be able to access a read-only version of your homebrew.\r\n\r\n## Helping out\r\nLike this tool? Want to buy me a beer? [Head here](https://www.patreon.com/stolksdorf) to help me keep the servers running.\r\n\r\nThis tool will **always** be free, never have ads, and I will never offer any \"premium\" features or whatever.\r\n\r\n\r\n\r\n>##### PDF Exporting\r\n> PDF Printing works best in Chrome. If you are having quality/consistency issues, try using Chrome to print instead.\r\n>\r\n> After clicking the \"Print\" item in the navbar a new page will open and a print dialog will pop-up.\r\n> * Set the **Destination** to \"Save as PDF\"\r\n> * Set **Paper Size** to \"Letter\"\r\n> * If you are printing on A4 paper, make sure to have the \"A4 page size snippet\" in your brew\r\n> * In **Options** make sure \"Background Images\" is selected.\r\n> * Hit print and enjoy! You're done!\r\n>\r\n> If you want to save ink or have a monochrome printer, add the **Ink Friendly** snippet to your brew before you print\r\n\r\n\r\n```\r\n```\r\n\r\n## Big things coming in v3.0.0\r\nWith the next major release of Homebrewery, v3.0.0, this tool *will no longer support raw HTML input for brew code*. All brews made previous to the release of v3.0.0 will still render normally.\r\n\r\n## New Things All The Time!\r\nWhat's new in the latest update? Check out the full changelog [here](/changelog)\r\n\r\n### Bugs, Issues, Suggestions?\r\nHave an idea of how to make The Homebrewery better? Or did you find something that wasn't quite right? Head [here](https://github.com/stolksdorf/homebrewery/issues/new) and let me know!.\r\n\r\n### Legal Junk\r\nThe Homebrewery is licensed using the [MIT License](https://github.com/stolksdorf/homebrewery/blob/master/license). Which means you are free to use The Homebrewery is any way that you want, except for claiming that you made it yourself.\r\n\r\nIf you wish to sell or in some way gain profit for what's created on this site, it's your responsibility to ensure you have the proper licenses/rights for any images or resources used.\r\n\r\n### More Resources\r\nIf you are looking for more 5e Homebrew resources check out [r/UnearthedArcana](https://www.reddit.com/r/UnearthedArcana/) and their list of useful resources [here](https://www.reddit.com/r/UnearthedArcana/comments/3uwxx9/resources_open_to_the_community/).\r\n\r\n\r\n\r\n<img src='http://i.imgur.com/hMna6G0.png' style='position:absolute;bottom:50px;right:30px;width:280px' />\r\n\r\n<div class='pageNumber'>1</div>\r\n<div class='footnote'>PART 1 | FANCINESS</div>\r\n\r\n\r\n\r\n\r\n\\page\r\n\r\n# Appendix\r\n\r\n### Not quite Markdown\r\nAlthough the Homebrewery uses Markdown, to get all the styling features from the PHB, we had to get a little creative. Some base HTML elements are not used as expected and I've had to include a few new keywords.\r\n\r\n___\r\n* **Horizontal Rules** are generally used to *modify* existing elements into a different style. For example, a horizontal rule before a blockquote will give it the style of a Monster Stat Block instead of a note.\r\n* **New Pages** are controlled by the author. It's impossible for the site to detect when the end of a page is reached, so indicate you'd like to start a new page, use the new page snippet to get the syntax.\r\n* **Code Blocks** are used only to indicate column breaks. Since they don't allow for styling within them, they weren't that useful to use.\r\n* **HTML** can be used to get *just* the right look for your homebrew. I've included some examples in the snippet icons above the editor.\r\n\r\n\r\n\r\n```\r\n```\r\n\r\n\r\n### Images\r\nImages must be hosted online somewhere, like imgur. You use the address to that image to reference it in your brew. Images can be included 'inline' with the text using Markdown-style images. However for background images more control is needed.\r\n\r\nBackground images should be included as HTML-style img tags. Using inline CSS you can precisely position your image where you'd like it to be. I have added both a inflow image snippet and a background image snippet to give you exmaples of how to do it.\r\n\r\n\r\n\r\n### Crediting Me\r\nIf you'd like to credit The Homebrewery in your brew, I'd be flattered! Just reference that you made it with The Homebrewery.\r\n\r\n\r\n\r\n<div class='pageNumber'>2</div>\r\n<div class='footnote'>PART 2 | BORING STUFF</div>\r\n\r\n\r\n","changelog":"# changelog\r\n\r\n### Saturday, 22/04/217 - v2.7.4\r\n- Give ability to hide the render warning notification\r\n\r\n\r\n### Friday, 03/03/2017 - v2.7.3\r\n- Increasing the range on the Partial Page Rendering for a quick-fix for it getting out of sync on long brews.\r\n\r\n### Saturday, 18/02/2017 - v2.7.2\r\n- Adding ability to delete a brew from the user page, incase the user creates a brew that makes the edit page unrender-able. (re:309)\r\n\r\n## BIG NEWS\r\nWith the next major release of Homebrewery, v3.0.0, this tool *will no longer support raw HTML input for brew code*. Most issues and errors users are having are because of this feature and it's become too taxing to help and fix these issues.\r\n\r\nAll brews made previous to the release of v3.0.0 will still render normally.\r\n\r\n### Thursday, 19/01/2017 - v2.7.0\r\n- Fixed saving multiple authors and multiple systems on brew metadata (thanks u/PalaNolho re:282)\r\n- Adding in line highlight for new pages\r\n- Added in a simple brew lookup for admin\r\n\r\n\r\n### Saturday, 14/01/2017 - v2.7.0\r\n- Added a new Render Warning overlay. It detects situations where the brew may not be rendering correctly (wrong browser, browser is zoomed in...) and let's the user know\r\n\r\n\r\n### Sunday, 25/12/2016 - v2.7.0\r\n- Switching over to using Vitreum v4\r\n - Removed gulp, all tasks are run through npm scripts\r\n- Updating docs for local dev\r\n- Removing support for Docker. I have never used it, nor will I ever test for it, so I don't want to continue to explictly support it on this repo. Feel free to make a fork and make it docker-able though :)\r\n- Changed icon for the metadata\r\n- Made links useable in footer (thanks u/Dustfinger1 re:249)\r\n- Added print media queries to remove box shadow on print (thanks u/dmmagic re: 246)\r\n- Fixed realtime renderer not functioning if loaded with malformed html on load (thanks u/RattiganIV re:247)\r\n- Removed a lot of unused files in shared\r\n- vitreum v4 now lets me use codemirror as a pure node dependacy\r\n\r\n\r\n\r\n### Saturday, 03/12/2016 - v2.6.0\r\n- Added report back to the edit page\r\n- Changed metaeditor icon\r\n- Added a button to quickly share your brew to reddit :)\r\n- Disabled Partial Page Rendering unless your brew hits 75 pages or longer\r\n- The brew renderer will now try and use your first page to judge the page size of each of your brews. This allows you now to set landscape and other weird sizes and everthing should work fine :)\r\n- UI on the user page improved (thanks u/PalaNolho)\r\n- Fixed lists not breaking across columns (thanks u/tyson-nw)\r\n- Added a table of contents snippet (thanks u/tullisar)\r\n- Added a multicolumn snippet\r\n\r\n\r\n\r\n### Thursday, 01/12/2016\r\n- Added in a snippet for a split table\r\n- Added an account nav item to new page\r\n\r\n\r\n### Sunday, 27/11/2016 - v2.5.1\r\n- Fixed the column rendering on the new user page. Really should have tested that better\r\n- Added a hover tooltip to fully read the brew description\r\n- Made the brew items take up only 25% allowing you to view more per row.\r\n\r\n### Wednesday, 23/11/2016 - v2.5.0\r\n- Metadata can now be added to brews\r\n- Added a metadata editor onto the edit and new pages\r\n- Moved deleting a brew into the metadata editor\r\n- Added in account middleware\r\n- Can now search for brews by a specific author\r\n- Editing a brew in anyway while logged in will now add you to the list of authors\r\n- Added a new user page to see others published brews, as well as all of your own brews.\r\n- Added a new nav item for accessing your profile and logging in\r\n\r\n\r\n### Monday, 14/11/2016\r\n- Updated snippet bar style\r\n- You can now print from a new page without saving\r\n- Added the ability to use ctrl+p and ctrl+s to print and save respectively.\r\n\r\n### Monday, 07/11/2016\r\n- Added final touches to the html validator and updating the rest of the branch\r\n- If anyone finds issues with the new HTML validator, please let me know. I hope this will bring a more consistent feel to Homebrewery rendering.\r\n\r\n### Friday, 09/09/2016 - v2.4.0\r\n- Adding in a HTML validator that will display warnings whenever you save. This should stop a lot of the issues generated with pages not showing up.\r\n\r\n### Saturday, 20/08/2016 - v2.3.0\r\n- Added in a license file\r\n- Updated the welcome text\r\n- Added in a much better Error page\r\n- If you visit a deleted brew, it will now remove it from your recent list. (Thanks u/sIllverback!)\r\n- Improved parsing of embedded html text in brews. (Thanks u/com-charizard!)\r\n- Added in a new coverpage snippet\r\n- Homebrewery will now try and onsert a good title for your brew if you don't provide one\r\n- Homebrewery now re-renders properly when you zoom\r\n- Fixed the noteblock overlapping into titles (thanks u/dsompura!)\r\n- Fixed a bad search route in the admin panel (thanks u/SnappyTom!)\r\n\r\n\r\n### Friday, 29/07/2016 - v2.2.7\r\n- Adding in descriptive note blocks. (Thanks calculuschild!)\r\n\r\n### Thursday, 07/07/2016 - v2.2.6\r\n- Added a new nav item on the homepage for accessing both recently viewed and edited brews (thanks [ChosenSeraph!](https://github.com/stolksdorf/homebrewery/issues/147))\r\n\r\n### Friday, 10/06/2016 - v2.2.4\r\n- Added an id to each rendered page\r\n- Allows adding in hyperlinks to specific pages\r\n- Even works after you print to pdf!\r\n\r\n### Tuesday, 07/06/2016 - v2.2.2\r\n- Fixed bug with new markdown lexer and aprser not working on print page\r\n\r\n### Sunday, 05/06/2016 - v2.2.1\r\n- Adding in a new Class table div block. The old Class table block used weird stacking of HTML elements, resulting is difficult to control behaviour and poor interactiosn with the rest of the page. This new block is much easier to style and work with.\r\n- Added in a new wide table snippet\r\n- Added in a new auto-incremeting page number snippet (thakns u/Ryrok!)\r\n- Lists in monster stat blocks should be fixed now\r\n\r\n\r\n### Saturday, 04/06/2016 - v2.2.0\r\n- MIgrating The Homebrewery over to hombrewery.naturalcrit.com. It know runs on it's own server, with it's own repo separate from the other tools I'm working on. Makes updating and deploying much easier.\r\n\r\n\\page\r\n\r\n### Sunday, 29/05/2016 - v2.1.0\r\n- Finally added a syntax for doing spell lists. A bit in-depth about why this took so long. Essentially I'm running out of syntax to use in stardard Markdown. There are too many unique elements in the PHB-style to be mapped. I solved this earlier by stacking certain elements together (eg. an `<hr>` before a `blockquote` turns it into moster state block), but those are getting unweildly. I would like to simply wrap these in `div`s with classes, but unfortunately Markdown stops processing when within HTML blocks. To get around this I wrote my own override to the Markdown parser and lexer to process Markdown within a simple div class wrapper. This should open the door for more unique syntaxes in the future. Big step!\r\n- Override Ctrl+P (and cmd+P) to launch to the print page. Many people try to just print either the editing or share page to get a PDF. While this dones;t make much sense, I do get a ton of issues about it. So now if you try to do this, it'll just bring you imediately to the print page. Everybody wins!\r\n- The onboarding flow has also been confusing a few users (Homepage -> new -> save -> edit page). If you edit the Homepage text now, a Call to Action to save your work will pop-up.\r\n- Added a 'Recently Edited' and 'Recently Viewed' nav item to the edit and share page respectively. Each will remember the last 8 items you edited or viewed and when you viewed it. Makes use of the new title attribute of brews to easy navigatation.\r\n- Paragraphs now indent properly after lists (thanks u/slitjen!)\r\n\r\n### Friday, 27/05/2016 - v2.0.6\r\n- Updated the issue template for (hopefully) better reporting\r\n- Added suggestion to use chrome while PDF printing\r\n\r\n### Wednesday, 25/05/2016 -v2.0.5\r\n- The class table generators have the proper ability score improvement progression.\r\n\r\n### Tuesday, 24/05/2016 - v2.0.4\r\n- Fixed extra wide monster stat blocks sometimes only being one column\r\n- The class table generators now follow the proper progression from the PHB (thakns u/IrishBandit)\r\n\r\n### Thursday, 19/05/2016 - v2.0.2\r\n\r\n- No longer server-side pre-render brews, just incase the user entered invalid HTML, it might crahsh the server\r\n- Bumped up the allowed entity size for extra-large brew (Thanks for reporting it dickboner93)\r\n- Added a little error box when a save fails with a custom link to reporting the issue on github.\r\n\r\n\\page\r\n\r\n### Saturday, 14/05/2016 - v2.0.0 (finally!)\r\n\r\nI've been working on v2 for a *very* long time. I want to thank you guys for being paitent.\r\nIt started rather small, but as more and more features were added, I decided to just wait until everything was polished.\r\n\r\nMassive changelog incoming:\r\n\r\n#### Brews\r\n- New flow for creating new brews. Before creating a new brew would immedaitely create a brew in the database and let you edit it. Many people would create a new brew just to experiment and close it, which lead to many \"abandoned brews\" (see the Under the hood section below). This started eating up a ton of database space. You now have to manually save a new brew for the first time, however Homebrewery will store anything you don't have saved in local storage, just in case your browser crashes or whatever, it will load that up when you go back to `/homebrew/new`\r\n- Black blocking edges around notes and stat blocks when printing to PDFs have been fixed\r\n- Borders sometimes not showing up in the second column have been fixed\r\n- All pseudo-element borders have been replaced with reliable border images.\r\n- Chrome can finally print to PDF as good as Chrome Canary! Updating instructions.\r\n- Added a little page number box.\r\n- Added in a new editable Brew Title. This will be shown in the navbar on share pages, and will default to the file name when you save as PDF. All exsisting brews will be defaulted with an empty title.\r\n- Mutliline lists render better now\r\n- Firefox rendering has been slithgly improved. Firefox and Chrome render things **slightly** differently, over the course of a brew, these little changes add up and lead to very noticable rendering differences between the browsers. I'm trying my best to get Firefox rendering better, but it's a difficult problem.\r\n- A bunch of you have wanted to donate to me. I am super flattered by that. I created a [patreon page](https://www.patreon.com/stolksdorf). If you feel like helping out, head here :)\r\n\r\n#### Under the Hood Stuff\r\n- Setup a proper staging environment. Will be using this for tests, and hopefully getting the community to help me test future versions\r\n- Server-side prerendering now much faster\r\n- Regular weekly database back-ups. Your brews are safe!\r\n- Database is now uniquely indexed on both editId and shareId, page loads/saving should be much faster\r\n- Improved Admin console. This helps me answer people's questions about issues with their brews\r\n- Added a whole querying/pagniation API that I can use for stats and answering questions\r\n- Clearing out \"Abandoned\" brews (smaller than a tweet and haven't been viewed for a week). These account for nearly a third of all stored brews.\r\n\r\n#### Interface\r\n- Added in a whole new editor with syntax highlighting for markdown\r\n- Built a splitpane! Remembers where you left the split in between sessions\r\n- Re-organized the snippets into a hierarchical groups. Should be much easier to find what you need\r\n- Partial page rendering is working. The Homebrewery will now only load the viewable pages, and any page with `<style>` tags on them. If you are working on a large brew you should notice *significant* performance improvements\r\n- Edit page saving interface has been improved significantly. Auto-saves after 3 seconds on inactivity, now allows user to save at anytime. Will stop the tab from closing witha pop-up if there are unsaved changes.\r\n- Navbar and overall style has been improved and spacing made more consistent\r\n- Elements under the hood are way more organized and should behaviour much more reliably in many sizes.\r\n- Source now opens to it's own route `/source/:sharedId` instead of just a window. Now easier to share, and won't be blocked by some browsers.\r\n- Print page now auto-opens print dialog. If you want to share your print page link, just remove the `?dialog=true` parameter and it won't open the dialog.\r\n\r\n\r\n\r\n\\page\r\n\r\n### Wednesday, 20/04/2016\r\n- A lot of admin improvements. Pagninated brew table\r\n- Added a searching and removing abandoned brew api endpoints (turns out about 40% of brews are shorter that a tweet!).\r\n- Fixed the require cache being cleared. Pages should render a bit faster now.\r\n- Pulled in `kkragenbrink`s fix for nested lists, Thanks!\r\n\r\n\r\n### Wednesday, 06/04/2016 - v1.4\r\n* Pages will now partially render. This should greatly speed up *very* large homebrews. The Homebreery will figure out which page you should be looking at and render that page, the page before, and the page after.\r\n* Zooming should be fixed. I've changed the font size units to be cm, which match the units of the page. Zooming in and out now look much better.\r\n\r\n\r\n### Monday, 29/02/2016 - v1.3.1\r\n* Removng the changelog button from the Share page\r\n* Added a A4 page size snippet (thanks \tguppy42!)\r\n* Added support for `<sup>` and `<sub>` tags (thanks crashinworld14!)\r\n\r\n### Saturday, 20/02/2016\r\n* Fixed h1 headers not going full width (thanks McToomin27)\r\n* Added a github issue template\r\n\r\n## v1.3.0\r\n\r\n### Friday, 19/02/2016\r\n* Improved the admin panel\r\n* Added ability to clear away old empty brews\r\n* Added delete button to the edit page\r\n* Added a dynamically updating changelog page! Nifty!\r\n* Added stlying for wide monster stat blocks and single column class tables\r\n* Added snippets for wide monster stat blocks and single column class tables\r\n\r\n### Tuesday, 16/02/2016\r\n* Paragraphs right after tables now indent (thanks LikeAJi6!)\r\n* Added a `@page` css rule to auto turn off margins when printing\r\n* Added a `page-break` property on each `.phb` page to properly page the pages up when exporting (thanks Jokefury!)\r\n* Improved first character rendering on Firefox\r\n* Improved table spacing a bit\r\n* Changed padding at page bottom for better fit and clipping of elements\r\n* Improved spacing for bold text (thanks nickpunt!)\r\n\r\n\r\n## v1.2.0\r\n\r\n### Sunday, 17/01/2016\r\n* Added a printer friendly snippet that injects some CSS to remove backbrounds and images\r\n* Adjusted the styling specific to spell blocks to give it tighter spacing\r\n* Added a changelog! How meta!\r\n\r\n## v1.1.0\r\n\r\n### Thursday, 14/01/2016\r\n* Added view source to see the markdown that made the page\r\n* Added print view\r\n* Fixed API issues that were causing the server to crash\r\n* Increased padding on table cells\r\n* Raw html now shows in view source\r\n\r\n\r\n## v1.0.0 - Release\r\n\r\n### Wednesday, 3/01/2016\r\n\r\n* Added `phb.standalone.css` plus a build system for creating it\r\n* Added page numbers and footer text\r\n* Page accent now flips each page\r\n\r\n","brew":{"version":1,"views":62,"lastViewed":"2017-08-21T20:28:05.122Z","updatedAt":"2016-07-13T05:41:02.085Z","createdAt":"2016-05-05T23:37:13.196Z","published":false,"authors":[],"systems":[],"tags":"","description":"","text":"<img src='http://iceboundgame.com/img/puzzle/alchemykit.png' style='position:absolute;bottom:310px;right:0px;width:420px' />\n<div style='position:absolute;bottom:310px;right:30px;width:200px';> Illustration used without permission<br>Copyright © 2013 Fastermind Games</div>\n\n## Poisoner's Kit\nA poisoner’s kit includes the vials, chemicals, and other equipment necessary for the creation of poisons. Proficiency with this kit lets you add your proficiency bonus to any ability checks you make to craft or use poisons.\n\n### Foraging Ingredients\nMake an Intelligence (Nature) check with a DC based on the method chosen from the table below (Proficiency\nwith the herbalist's kit applies to this check if the\ncharacter doesn’t have proficiency in Nature.) This check can be made once per hour.\n\n##### Foraging Results\n| DC | Method of Search \n|:---|:-------------\n| 10 | Dedicated search without traveling\n| 15 | Traveling at a slow or stealthy pace \n| 18 | Traveling at a normal pace \n| 21 | Traveling at a fast pace\n\nOn a success, you recover one ingredient from the Foraging Ingredient table at the end of this section. On a failure, you find nothing. If you roll a 20 on your check, you recover one ingredient from the Special Ingredients table at the end of this section.\n\n### Crafting a Poison\nOnce you recover an ingredient, you can begin the process of crafting it into a poison. Crafting a poison requires an hour of work followed by an Intelligence Check using your poisoner’s kit, with a DC established by the difficulty modifiers of your added ingredients. You may craft a poison with no more than 5 ingredients. Any poison with a total DC less than 13 may be crafted in the field during a short rest. Otherwise crafting a poison must be done in a safe, workshop environment such as a room at an inn over the course of an hour.\n\nOn a successful crafting check, you create one vial of poison. On a failure, you do not create the poison and all used ingredients are lost. Should you roll a natural 20 on your crafting check, you are able to stretch the ingredients and make 2 vials. Vials are usable for 1 week after crafting before they decay.\n\n### Using Poisons \nYou may use an action to apply poison to a single melee weapon or 3 pieces of ammunition. Applied poison lasts for one minute and a single applied dose can harm a creature twice before depleting its potency. A single item may not have more than one dose applied to it at once.\n\nA creature hit by a poisoned weapon or ammunition must make a Constitution Saving Throw. The save DC is equal to 8 + the proficiency of the craftsman. On a failure, the creature is poisoned for one minute and suffers the effects of any of its ingredients. A poisoned creature must repeat the saving throw at the start of each of its turns.\n\n```\n```\n\n##### Foraging Ingredient Table\n|2d6| Ingredient | Effects | DC\n|:-:|:-----------------|:-----------------------|:--:\n| 2 |Mandrake Root |Increase Save DC by 2 | +2\n| 3 |Quicksilver Lichen|Deal 1d4 poison damage | +2\n| 4 |Milkweed Seeds |Reduce Crafting DC | -1\n| 5 |Wild Sageroot |Deal 2d4 poison damage | +3\n|6-8|Wyrmtongue |Required for all poisons| 10\n| 9 |Wild Sageroot |Deal 2d4 poison damage | +3\n|10 |Milkweed Seeds |Reduce Crafting DC | -1\n|11 |Quicksilver Lichen|Deal 1d4 poison damage | +2\n|12 |Mandrake Root |Increase Save DC by 2 | +2\n\n##### Special Ingredient Table\n|1d6| Ingredient | Effects | DC\n|:-:|:-------------|:---------------------------|:--:\n| 1 |Wrackwort |Target is blinded |+3\n| 2 |Spineflower |Double the damage dice |+3\n| 3 |Amanita Cap |Can't make any sound |+2\n| 4 |Emetic Wax |Target is Incapacitated |+1\n| 5 |Verdant Nettle|Target's speed becomes 0 |+2\n| 6 |Bloodleaf |Poison damage dice become d6|+3\n\n<div style='margin-top:320px'></div>\n\n> ##### Variant: Drow Poison\n> A Dark Elf with proficiency in poisoner’s kit may craft drow poison by imbuing a poison with their innate magical abilities. You consume all uses of your innate magic until the next dawn, imbuing your poison with Drow magic. \n\n> Drow Magic is treated as an ingredient with no modification to the crafting DC. It's effect is to cause a poisoned creature who failed their Constitution saving throw by 5 or more to fall unconcious. The poisoned creature wakes up if it takes damage or if another creature takes an action to shake it awake.\n\n<div class='pageNumber'></div>\n<div class='footnote'>POISONER'S KIT | Rules by /u/Shylocv </div>\n","title":"Poisoner's Kit","shareId":"ByZ4K8tb"},"account":{"username":"Modnar","issued":"2017-03-24T03:21:28.939Z"}});

**Comment by calculuschild** [comment]
Duplicate of #655
